jupyter/trustyai/ubi9-python-3.9/test/test_notebook.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "df03ef8f-cfd8-42d6-8d37-28c92ecb63b0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "EEE..\n",
+      "======================================================================\n",
+      "ERROR: test_Disparateimpact (__main__.TestTrustyaiNotebook)\n",
+      "----------------------------------------------------------------------\n",
+      "Traceback (most recent call last):\n",
+      "  File \"/tmp/ipykernel_86/2919058814.py\", line 29, in test_Disparateimpact\n",
+      "    score = disparate_impact_ratio_model(samples=X,\n",
+      "NameError: name 'X' is not defined\n",
+      "\n",
+      "======================================================================\n",
+      "ERROR: test_adverage (__main__.TestTrustyaiNotebook)\n",
+      "----------------------------------------------------------------------\n",
+      "Traceback (most recent call last):\n",
+      "  File \"/tmp/ipykernel_86/2919058814.py\", line 36, in test_adverage\n",
+      "    score = average_odds_difference(test=bias,\n",
+      "NameError: name 'bias' is not defined\n",
+      "\n",
+      "======================================================================\n",
+      "ERROR: test_fairnessmetrics (__main__.TestTrustyaiNotebook)\n",
+      "----------------------------------------------------------------------\n",
+      "Traceback (most recent call last):\n",
+      "  File \"/tmp/ipykernel_86/2919058814.py\", line 22, in test_fairnessmetrics\n",
+      "    score = average_odds_difference_model(samples=X,\n",
+      "NameError: name 'X' is not defined\n",
+      "\n",
+      "----------------------------------------------------------------------\n",
+      "Ran 5 tests in 0.007s\n",
+      "\n",
+      "FAILED (errors=3)\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<unittest.runner.TextTestResult run=5 errors=3 failures=0>"
+      ]
+     },
+     "execution_count": 37,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```

Comment:
I don't understand this deeply yet, but suppose that we can left the outputs field empty?

```suggestion
   "outputs": [],
```